tests/unit_tests/navier_stokes_unit_test/reynolds_averaged_navier_stokes_sa_neg_manufactured_solution_source.cpp

```diff
@@ -0,0 +1,145 @@
+#include <iomanip>
+#include <cmath>
+#include <limits>
+#include <type_traits>
+
+#include "ADTypes.hpp"
+
+#include <assert.h>
+#include <deal.II/base/types.h>
+#include <deal.II/grid/grid_generator.h>
+
+#include "physics/model.h"
+#include "assert_compare_array.h"
+#include "parameters/parameters.h"
+#include "physics/reynolds_averaged_navier_stokes.h"
+#include "physics/negative_spalart_allmaras_rans_model.h"
+
```

Comment:
this test is largely a copy paste from `unit_tests/navier_stokes_unit_test/navier_stokes_manufactured_solution_source.cpp` with the difference of calling a different physics object. Might be better to have some sort of input variable that switches the original file between RANS and NS; then in the CMakeList.txt, you pass the appropriate argument when you define the ctest's "COMMAND"

Reply:
would also make any future extensions to the test easier

Reply:
Open a new issue about this comment:
https://github.com/dougshidong/PHiLiP/issues/184#issue-1571763299